src/less/select/ds6/select-base.less

```diff
@@ -0,0 +1,46 @@
+.select {
+    display: inline-block;
+    height: 40px;
+    position: relative;
+}
+
+.select > select {
+    appearance: none;
+    background-color: #ffffff;
+    border: 1px solid #c7c7c7;
+    border-radius: 0;
+    font-size: 16px;
+    height: 40px;
+    padding: 8px 16px;
+    vertical-align: top;
+
+    &:focus {
+        border-color: @ds6-color-p004-blue;
+        outline: none;
+    }
+}
+
+.select.select__arrow > select {
```

Comment:
Just checking that the intention here with the tag selector is to enforce a11y? Like we do with buttons and links.

Reply:
Yes, trying to enforce the use of `<select>`.